2020-07-21 20:44:01 - root - INFO - Connection with MYSQL ready
2020-07-21 20:44:01 - root - INFO - Database was created successfully
2020-07-21 20:44:12 - root - INFO - Tables were created successfully
2020-07-21 20:45:34 - root - INFO - Browser connect to URL
2020-07-21 20:45:37 - root - INFO - Search for job title
2020-07-21 20:45:38 - root - INFO - Search for job location
2020-07-21 20:45:46 - root - ERROR - No popup to close
Traceback (most recent call last):
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/scraper.py", line 29, in close_popup
    pop_up = self.browser.find_element_by_xpath(self.config['Path']['POP_UP_XPATH'])
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //div[contains(@class,'modal_main')]//span[@class='SVGInline modal_closeIcon']

2020-07-21 20:46:48 - root - INFO - Browser connect to new URL
2020-07-21 20:47:04 - root - ERROR - No popup to close
Traceback (most recent call last):
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/scraper.py", line 29, in close_popup
    pop_up = self.browser.find_element_by_xpath(self.config['Path']['POP_UP_XPATH'])
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/Sheryl/PycharmProjects/Itc_data_mining/data_mining/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //div[contains(@class,'modal_main')]//span[@class='SVGInline modal_closeIcon']

2020-07-21 20:47:04 - root - CRITICAL - There is a problem with your jobs search keywords so can't get the number of pages
